### PR TITLE
terraform: Require the libvirt provider

### PR DESCRIPTION
**Pull request by tjoshum**
We need this field so that we work with terraform v0.13

### CODE REVIEW

**Review by paulyoong — APPROVED** [review]

**Comment by blaisedias** [comment]
Does this work with terraform versions < 0.13?

**Comment by tjoshum** [comment]
I think you have v0.12? Could you run terraform validate for me, rather than me trying to downgrade?

**Comment by blaisedias** [comment]
> I think you have v0.12? Could you run terraform validate for me, rather than me trying to downgrade?
Sorry, I moved to 0.13 ages ago, and have this change locally, but didn't check it in cause I was concerned that it would break earlier versions.

**Comment by gila** [comment]
I would argue that most people can figure out how to install terraform properly if they consider to use it. The alternative is, that we provide a version in our shell.nix which I think would be overkill. IOW, im fine with this as it is.

**Comment by blaisedias** [comment]
@tjoshum please can you update `terraform/README.adoc` to say that the files work with 0.13 or words to to that effect.

**Review by blaisedias — APPROVED** [review]

File: terraform/mod/libvirt/main.tf [hunk 1]
```diff
@@ -148,4 +148,11 @@ output "node_list" {
 
 terraform {
   required_version = ">= 0.12"
```
Comment (jonathan-teh) [inline terraform/mod/libvirt/main.tf:150]: Suggest bumping this up to 0.13.

Comment (blaisedias) [reply terraform/mod/libvirt/main.tf]: Done.

**Comment by blaisedias** [comment]
bors try

**Review by jonathan-teh — APPROVED** [review]

**Comment by blaisedias** [comment]
bors merge

**Pull request merged**